pdr_backend/sim/sim_plotter.py

```diff
@@ -24,7 +25,20 @@ def __init__(
         self.st = None
         self.aimodel_plotdata = None
 
-    def load_state(self):
+    @staticmethod
+    def available_snapshots():
```

Comment:
checked this against pickle/save code in sim_engine, and it looks good

i check test_sim_plotter.py and it's looking empty
we might want to just a dd a bit of coverage here in the future